go/paddlecloud/restclient.go

```diff
@@ -14,7 +14,15 @@ import (
 // HTTPOK is ok status of http api call
 const HTTPOK = "200 OK"
 
-var httpTransport = &http.Transport{}
+type RestClient struct {
```

Comment:
What are the benefits of adding a struct? We can not re-use the connection to the server if we need to create a new `RestClient` every time.